extendr-macros/tests/trybuild.rs

```diff
@@ -0,0 +1,5 @@
+#[test]
+fn test_macro_failures() {
+    let t = trybuild::TestCases::new();
+    t.compile_fail("tests/cases/*.rs");
```

Comment:
Hm, can this do macros expand? Maybe we can use it for snapshot testing macros, instead of doing it through `extendrtests`?

Reply:
This is the rust way of making these tests. I don't think looping in the weird snapshot tests through R should replace this.

Reply:
In my understanding, these cannot be expanded because the macros usages are wrong and fail.

Reply:
I now understand what you mean. This is not possible to do with testthat as the snapshot tests are expected to succeed, and not fail, like these tests. This is now resolved.